downstairs/src/lib.rs

```diff
@@ -1648,31 +1647,7 @@ impl ActiveConnection {
                 // offset); we inject that extra information in this terrible
                 // match statement.
```

Comment:
is this comment now out of date? :)

Reply:
Fixed in c1d472b8361f; we can also replace `r.blocks.clone()` with `r.blocks` 😎